.pre-commit-config.yaml

```diff
@@ -0,0 +1,146 @@
+# A config file for pre-commit.com
+#
+# Install pre-commit.com: https://pre-commit.com/#install
+#
+# Then run the following line to set up a pre-commit hook:
+# pre-commit install
+#
+# And this to update hooks to the newest version:
+# pre-commit autoupdate
+#
+# Now, whenever you try to commit, hooks defined in this file would run
+# and check your files.  They would run only only on staged files that
+# had changes.
+#
+# You can manuall trigger the hooks like this:
+# pre-commit run
+#
+# You can tell it to check specific files:
+# pre-commit run --files *.ts
+#
+# Or give it all files:
+# pre-commit run --all-files
```

Comment:
Is there a way to run it just for the output without updating any files?

Reply:
I don't think there is such parameter.
One solution would be to run `git stash && git stash apply` to save a copy of the working tree into stash. Then, after running pre-commit you can recover the previous state by running `git checkout stash -- .`